linux/deploy_vm/ubuntu/prepare_ubuntu_iso_install.yml

```diff
@@ -28,7 +28,19 @@
     unattend_install_conf: "Ubuntu/Server/user-data.j2"
   when: ubuntu_install_method is defined and ubuntu_install_method == "cloud-init"
```

Comment:
I suggest you putting line 17 ~ line 29 into a block for Ubuntu server.

Reply:
change it. Thanks